README.md

```diff
@@ -1,79 +1,114 @@
-Takeaway Challenge
-==================
+# Takeaway Challenge #
```

Comment:
I forgot to do a README again! 🤦‍♂️ 
Well done on this comprehensive overview! Ill be taking notes from you when i remember to write a README next time!